client/clientset/versioned/fake/register.go

```diff
@@ -34,14 +37,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 // AddToScheme adds all types of this clientset into the given scheme. This allows composition
 // of clientsets, like in:
 //
-//   import (
-//     "k8s.io/client-go/kubernetes"
-//     clientsetscheme "k8s.io/client-go/kubernetes/scheme"
-//     aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
-//   )
+//	import (
```

Comment:
@kbowers-ibm maybe this is formatting is unwanted

Reply:
Yeah potentially, I've tried to change it, but I can't manually edit those files and when I run make before-pr it changes the whitespace there for some reason

Reply:
don't worry, just pointed it out just in case there was a reason for this. I'm ok with it.